packages/core/admin/ee/admin/src/pages/SettingsPage/pages/ApplicationInfoPage/components/tests/AdminSeatInfo.test.tsx

```diff
@@ -93,19 +93,4 @@ describe('<AdminSeatInfo />', () => {
       'https://cloud.strapi.io/profile/billing'
     );
   });
-
-  test('Render OVER_LIMIT icon', () => {
```

Comment:
Why remove this test?

Reply:
It wasn't testing what we assumed it was testing, it was testing a tooltip being in the DOM, but by default they shouldn't be until their envoked.